README.md

```diff
@@ -15,6 +15,17 @@ clasp login
 Create an Apps Script script, if you don't already have one you want to use.
 - You probably want a [container-bound script](https://developers.google.com/apps-script/guides/bound#access_to_bound_scripts) linked to the OneStop spreadsheet.
 
+Make your own copy of the OneStop here: https://drive.google.com/drive/folders/1s_u-hmstlLL1JqRKyKJ7r2AMyyTCCiut
+
+Get the Script ID for your personal dev OneStop by following these steps:
+- Open your new sheet
+- Click Extensions > App Script
+- Select the first project that appears
```

Comment:
this should probably be more specific - if you copied josh wong's or mine it happens to be the first project is fine, but if they directly copied the real onestop it doesn't exist. maybe they could just directly create their own project